Netconf-notif-v12-examples-v17Sept2018/ietf-subscribed-notifications.yang

```diff
@@ -543,9 +543,10 @@ module ietf-subscribed-notifications {
         description
           "Adds additional objects which can be modified by RPC.";
         leaf stream {
-          type stream-ref {
-            require-instance false;
-          }
+	  type string;
```

Comment:
Changed to allow `yanglint` to run clean.

Reply:
Should we be modifying the YANG model to address possible bugs in yanglint?

Reply:
It really is something which should be fixed in YANGLINT.   The streams are 'ro' because they can't be configured by users.  Perhaps YANGLINT assumes the information must therefore be coming from the operational datastore.   But certainly configured subscriptions might want to refer to them.    This type of design pattern is seen in several places.  YANGLINT shouldn't preclude this.
https://www.ietf.org/mail-archive/web/netconf/current/msg11244.html 
https://www.ietf.org/mail-archive/web/netconf/current/msg12408.html